docs/docsgen/source/requirements.txt

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
 
```

Comment:
Probably fine that not to add license in requirements.txt?

Reply:
In general, every file should have these two information, but I can also add that to the .reuse/dep5 file. 

I'm not quite clear yet what best practices would be desired or fit for us?

What exactly don't you like about the addition? 

For other points I'm also not sure yet how to deal with it, e.g. if documentation is automatically generated from it. 

I'll remove that change from the commit.

Reply:
I think I would add it everywhere where the license is clear, so that it is from us..., and does not collide with any functionality. For non-trivial files like __init__.py you could of course also choose cc0? Generated files where the "reuse checker" does not work. I would also exclude and list in the dep5 file. I would have given the header to the whole pipelines... But of course you can leave it aside in the first round and discuss it later.  Why would you exclude CI related files? You mean excluding from putting a header / comment to the top? or mark with the licence and copyright in dep5?

Reply:
> Why would you exclude CI related files? You mean excluding from putting a header / comment to the top? or mark with the licence and copyright in dep5?

I just feel strictly CI related files are not part of source code. At least exclude them from putting a header / comment to the top, I think. I am fine with having them in dep5.

Reply:
ok, we can start this way. I added some directories to the dep5 file.